Course.java

```diff
@@ -0,0 +1,47 @@
+public class Course {
+  private String courseName;
+  public int classSize =0;
+  private ArrayList<Person> StudentList;
+  private ArrayList<Person> mentorList;
```

Comment:
nit: line break after the private variables for readability

Reply:
nit: The variable name should probably be pluralize. `mentorsList` same for students.